ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__weekly_trigger.yaml

```diff
@@ -0,0 +1,33 @@
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
Do you think we should add firewatch to this, or should we just rely on Slack notifications? I lean towards Jira so that someone has an actionable task to work on when this fails.